app/src/main/java/org/mozilla/focus/browser/cursor/CursorView.kt

```diff
@@ -33,23 +29,13 @@ private val HIDE_AFTER_MILLIS = TimeUnit.SECONDS.toMillis(3)
 class CursorView(context: Context, attrs: AttributeSet) : View(context, attrs) {
 
     private val hideHandler = CursorHideHandler(this)
-
-    // The radius is half the size of the container its drawn in.
-    private val radius = context.resources.getDimensionPixelSize(R.dimen.remote_cursor_size) / 2f
-    private val paint = Paint().apply {
-        style = Paint.Style.FILL
-        alpha = CURSOR_ALPHA
-        isAntiAlias = true
-        shader = RadialGradient(radius, radius, /* radius */ radius,
-                ContextCompat.getColor(context, R.color.teal50), ContextCompat.getColor(context, R.color.photonBlue50),
-                Shader.TileMode.CLAMP)
-    }
+    private val CROSSHAIR_OFFSET = 6
 
     @UiThread
     fun updatePosition(x: Float, y: Float) {
-        // In onDraw, we offset the initial position from the origin: we undo that offset here.
-        translationX = x - radius
-        translationY = y - radius
+        // Offset the position to better align with the crosshairs.
+        translationX = x - CROSSHAIR_OFFSET
```

Comment:
Relatedly, now that the click point is *not* the center of the image, is clicking accurate in relation to the crosshairs on the image?

Reply:
_Yes_. See further explanation in the other comment. I might be able to get it more accurate if I spend some more time on it